storymap/es/index.html

```diff
@@ -0,0 +1,936 @@
+                caption.setAttribute("class", "caption");
+                caption.innerHTML = `<em>${record.caption}</em>`;
+                chapter.appendChild(caption);
+            }
+
+            if (record.description) {
+                var story = document.createElement('p');
+                story.innerHTML = record.description;
+                chapter.appendChild(story);
+
+                // add author to chapters if found in config
+                if (record.author) {
+                    author = document.createElement('p');
+                    author.innerHTML = '<strong>Por ' + record.author + '</strong>';
+                    chapter.appendChild(author);
+                }
+
+                // add website to chapters if found in config
+                if (record.website) {
+                    var website = document.createElement('p');
+                    website.innerHTML = `» ${record.website}`;
+                    chapter.appendChild(website);
+                }
+
+                // add return to top link to each chapter
+                var returnTop = document.createElement('p');
+                returnTop.setAttribute("class", "returntop");
+                returnTop.innerHTML = '<a href="#header">Volver arriba</a>';
+                chapter.appendChild(returnTop);
+            }
+
+            // add legend to chapters if found in config
+            if (record.legend) {
+                var legend = document.createElement('div');
+                legend.setAttribute("class", "legend");
+                legend.setAttribute("id", (record.id + "legend"));
+                legend.innerHTML = '<div>' + record.legend + '</div>';
+                document.body.append(legend);
+            }
+
+            container.setAttribute('id', record.id);
+            container.classList.add('step');
+            if (idx === 0) {
+                container.classList.add('active');
+            }
+
+            chapter.classList.add(config.theme);
+            container.appendChild(chapter);
+            container.classList.add(alignments[record.alignment] || 'centered');
+            if (record.hidden) {
+                container.classList.add('hidden');
+            }
+            features.appendChild(container);
+        });
+
+        story.appendChild(features);
+
+        var footer = document.createElement('div');
+
+        if (config.translation) {
+            var button = document.createElement('div');
+            switch(config.translation) {
+                case 'es':
+                    button.innerHTML = '<a href="https://digital-democracy.org/storymap/es/"><button id="button">Español</button></a>';
+                break;
+                case 'en':
+                    button.innerHTML = '<a href="https://digital-democracy.org/storymap/"><button id="button">English</button></a>';
+                break
+            }
+            header.appendChild(button);
```

Comment:
Since spanish is not ready, should we get rid of this and not have the button available?